wordle-web/pages/index.vue

```diff
@@ -1,39 +1,71 @@
 <template>
-  <v-container>
-    <v-card>
-      <v-card-title>Wordle</v-card-title>
+  <v-container class="myFontDefault">
+    <v-card class="text-center">
+      <v-alert v-if="game.gameState != GameState.Playing" :color="game.gameState == GameState.Won ? 'success' : 'error'"
+        class="mb-5" tile>
+        <h3>
+          You've
+          {{ game.gameState == GameState.Won ? "Won" : "Lost" }}
+        </h3>
+        <v-card-text>
+          The word was: <strong>{{ game.secretWord }}</strong>
+        </v-card-text>
+        <v-btn variant="outlined" @click="game.startNewGame()">
+          <v-icon size="large" class="mr-2"> mdi-restart </v-icon> Restart Game
+        </v-btn>
+      </v-alert>
+      <v-card-title v-else>Wordle</v-card-title>
+
+      <GameBoardGuess v-for="(guess, i) of game.guesses" :key="i" :guess="guess" />
 
-      <div v-if="game.gameState == GameState.Won">
-        You've Won!
+      <div class="my-5">
+        <ValidWord />
       </div>
-      <div v-if="game.gameState == GameState.Lost">
-        You've Lost!
+      
+        <!--<h3>
+          <v-menu>
+          <template v-slot:activator="{ props }">
+            <v-btn v-bind="props" icon>
+              <v-icon>mdi-view-list</v-icon>
+            </v-btn>
+          </template>
+          <v-list>
+            <v-list-item v-for="word in validWords" :key="word">
+              <v-list-item-title>{{ word }}</v-list-item-title>
+            </v-list-item>
+          </v-list>
+        </v-menu>
+      </h3>
+      -->
```

Comment:
I strongly recommend against leaving logic that has been commented out in final code. It's not such a big deal in this assignment/class, but it's a good habit to get into for industry.